package.json

```diff
@@ -14,7 +15,7 @@
     "prepare": "husky",
     "dev": "cross-env NODE_ENV=development webpack serve --config src/webpack.config.ts",
     "build": "webpack --config src/webpack.config.ts",
-    "lint:eslint": "eslint src --ext .js,.jsx,.ts,.tsx",
+    "lint:eslint": "eslint src 'src/**/*.{js,jsx,ts,tsx,css,json,html}'",
```

Comment:
We seem to now be specifying `src` twice, is that correct?

Reply:
Oh oops, no that isn't. Will fix